src/Test/Perf/Utilities/TraceManager.cs

```diff
@@ -31,31 +30,24 @@ public static ITraceManager NoOpTraceManager()
 
     public class TraceManager : ITraceManager
     {
-        private readonly ScenarioGenerator _scenarioGenerator;
+        private ScenarioGenerator _scenarioGenerator;
         private readonly string _cpcPath;
 
         private int _startEventAbsoluteInstance = 1;
         private int _stopEventAbsoluteInstance = 1;
 
-        public TraceManager(
-            string cpcPath,
-            string scenarioPath) : base()
+        public TraceManager(string cpcPath) : base()
         {
             _cpcPath = cpcPath;
-            _scenarioGenerator = new ScenarioGenerator(scenarioPath);
+            _scenarioGenerator = new ScenarioGenerator();
         }
 
-        public bool HasWarmUpIteration
-        {
-            get
-            {
-                return true;
-            }
-        }
+        public bool HasWarmUpIteration => true;
 
         // Cleanup the results directory and files before every run
         public void Initialize()
         {
+            /*
```

Comment:
If we dont cleanup these files then traces collected during the previous runs will become part of the current run. `ConsumptionTempResults.xml` is appended with trace info